### PR TITLE
fix(client): printStack for Node-API

### PR DESCRIPTION
**Pull request by williamluke4**

### CODE REVIEW

File: src/packages/client/src/runtime/utils/printStack.ts [hunk 1]
```diff
@@ -77,13 +77,16 @@ function parseStack({
   const trace = stack.reverse().find((t) => {
     return (
       t.file &&
+      t.file !== '<anonymous>' &&
       !t.file.includes('@prisma') &&
       !t.file.includes('getPrismaClient') &&
+      !t.file.startsWith('internal/') &&
       !t.methodName.includes('new ') &&
       !t.methodName.includes('_getCallsite') &&
       t.methodName.split('.').length < 4
```
Comment (Jolg42) [inline src/packages/client/src/runtime/utils/printStack.ts:86]: These conditions probably all make sense somehow but they are a bit mysterious 🤔 

What is `t.file !== '<anonymous>'`? a Node-API thing only maybe?

And what is `!t.file.startsWith('internal/') &&` I don't remember files starting like this 🤔

Comment (millsp) [reply src/packages/client/src/runtime/utils/printStack.ts:86]: Would also love to know this, @williamluke4

Comment (williamluke4) [reply src/packages/client/src/runtime/utils/printStack.ts:86]: So what I can gather is that `internal/` are internal nodejs files that are returned in the stack trace and `<anonymous>` seems to happen when there is a `try catch`.

**Review by Jolg42 — APPROVED** [review]

**Comment by janpio** [comment]
I thought this would fix this issue, https://github.com/prisma/prisma/issues/7874, but seems it is still present even after this PR has been merged 😭 https://github.com/prisma/prisma/issues/7874#issuecomment-873260363

**Pull request merged**